integration/js/utils/SdkBaseTest.js

```diff
@@ -300,6 +300,14 @@ class SdkBaseTest extends KiteBaseTest {
         return 'meeting'
     };
   }
+
+  getTransformedURL = (url, key, value) => {
+    let appendOption = '?';
+    if (url && url.indexOf('?') !== -1) {
+      appendOption = '&'
+    }
+    return url + appendOption + key + '=' + value;
```

Comment:
There's a bug in this.

I'd write it this way:

```
getTransformedURL = (url, key, value) => {
  const sep = url.includes('?') ? '&' : '?';
  return `${url}${sep}${encodeURIComponent(key)}=${encodeURIComponent(value)}`;
}
```

Reply:
Thanks, updated